From: Christian Kastner <debian@kvr.at>
Date: Mon, 26 May 2014 00:44:59 +0200
Subject: Disable some tests

Some tests do not properly check whether features they need are available.
These will be fixed in the next upstream release.

Forwarded: no
Last-Update: 2014-05-25

---
 tests/bugzillas/bz1031154/runtest.sh | 87 ------------------------------------
 tests/keyctl/padd/useradd/runtest.sh | 12 -----
 2 files changed, 99 deletions(-)
 delete mode 100644 tests/bugzillas/bz1031154/runtest.sh

diff --git a/tests/bugzillas/bz1031154/runtest.sh b/tests/bugzillas/bz1031154/runtest.sh
deleted file mode 100644
index e4d64d4..0000000
--- a/tests/bugzillas/bz1031154/runtest.sh
+++ /dev/null
@@ -1,87 +0,0 @@
-#!/bin/sh
-
-# Test for https://bugzilla.redhat.com/show_bug.cgi?id=1031154
-
-. ../../prepare.inc.sh
-. ../../toolbox.inc.sh
-
-
-# ---- do the actual testing ----
-
-result=PASS
-echo "++++ BEGINNING TEST" >$OUTPUTFILE
-
-# we need a reference time to scan the audit log from so as not to pick up old
-# results from this test.
-base_date=`date +"%x@%X"`
-base_time=${base_date#*@}
-base_date=${base_date%@*}
-sleep 1
-
-# reset the permissive audit log autocancel thing
-load_policy
-
-# we need to be in permissive mode
-marker "ENTER SELINUX PERMISSIVE MODE"
-
-mode=`getenforce`
-
-if [ "$mode" != "Permissive" ]
-then
-	echo setenforce Permissive >>$OUTPUTFILE
-	if ! setenforce Permissive
-	then
-		failed
-	fi
-fi
-
-# create a big key to probe
-marker "CREATE BIG KEY"
-pcreate_key_by_size 8192 big_key test-key @s
-expect_keyid id
-
-# check the big key is file backed and the right size
-marker "CHECK BIG KEY"
-xid=`printf %08x $id`
-
-pk=`cat /proc/keys | grep "^$xid.*test-key: 8192 \\[file\\]"`
-echo $pk >>$OUTPUTFILE
-if [ -z "$pk" ]
-then
-	echo "+++ Incorrectly created key" >>$OUTPUTFILE
-	cat /proc/keys | grep "^$xid" >>$OUTPUTFILE
-	failed
-fi
-
-# use a separate context to access the key
-marker "ACCESS INTERCONTEXT"
-
-echo runcon system_u:system_r:httpd_t:s0-s0:c0.c1023 keyctl print $id >>$OUTPUTFILE
-if ! runcon system_u:system_r:httpd_t:s0-s0:c0.c1023 keyctl print $id >/dev/null 2>>$OUTPUTFILE
-then
-	failed
-fi
-
-# examine the audit logs
-marker "EXAMINE AUDIT LOGS"
-
-echo ausearch -m AVC -i --subject httpd_t -ts $base_date $base_time \| audit2allow \| grep '-P "allow httpd_t user_tmpfs_t:file [{] (open |read )+[}];"' >>$OUTPUTFILE
-if ausearch -m AVC -i --subject httpd_t -ts $base_date $base_time 2>>$OUTPUTFILE | audit2allow 2>>$OUTPUTFILE | grep -P "allow httpd_t user_tmpfs_t:file [{] (open |read )+[}];"
-then
-	failed
-fi
-
-marker "RESTORE SELINUX MODE"
-if [ "$mode" != "Permissive" ]
-then
-	echo setenforce $mode >>$OUTPUTFILE
-	if ! setenforce $mode
-	then
-		failed
-	fi
-fi
-
-echo "++++ FINISHED TEST: $result" >>$OUTPUTFILE
-
-# --- then report the results in the database ---
-toolbox_report_result $TEST $result
diff --git a/tests/keyctl/padd/useradd/runtest.sh b/tests/keyctl/padd/useradd/runtest.sh
index 94c9609..d5e4db9 100644
--- a/tests/keyctl/padd/useradd/runtest.sh
+++ b/tests/keyctl/padd/useradd/runtest.sh
@@ -56,18 +56,6 @@ then
 	expect_keyid keyid
 	md5sum_key $keyid
 	expect_payload payload "f128f774ede3fe931e7c6745c4292f40"
-
-	marker "ADD SMALL BIG KEY"
-	pcreate_key_by_size 128 big_key small @s
-	expect_keyid keyid
-	md5sum_key $keyid
-	expect_payload payload "f09f35a5637839458e462e6350ecbce4"
-
-	marker "ADD HUGE BIG KEY"
-	pcreate_key_by_size $((1024*1024-1)) big_key huge @s
-	expect_keyid keyid
-	md5sum_key $keyid
-	expect_payload payload "e57598cd670284cf7d09e16ed9d4b2ac"
 fi
 
 marker "CLEAR KEYRING"
-- 
2.0.0.rc4

